doc/changes/latest.rst

```diff
@@ -20,6 +20,8 @@ Enhancements
 
 - Update Decoder objects to use the more efficient ``LogisticRegressionCV`` (:gh:`3736` by `Michelle Wang`_).
 
+- [DOC] Inconsistent description of return keys for fetch_atlas_surf_destrieux (:gh:`3774` by `Tarun Samanta`_).
```

Comment:
This needs to be a description of what you did rather than the title of the PR. Something like:

```suggestion
- Make return key names in the description file of destrieux surface consistent with :func:`~datasets.fetch_atlas_surf_destrieux` (:gh:`3774` by `Tarun Samanta`_).
```

The syntax I use for the function name is just to link to the API in the rendered doc